--- a/rust/platform/triple.bzl
+++ b/rust/platform/triple.bzl
@@ -21,7 +21,7 @@
             - abi (str, optional): The abi to use or None if abi does not apply.
             - str (str): Original string representation of the triple
     """
-    if triple in ("wasm32-wasi", "wasm32-wasip1", "wasm32-wasip2"):
+    if triple in ("wasm32-wasi", "wasm32-wasip1", "wasm32-wasip1-threads", "wasm32-wasip2"):
         trip = triple
         if trip == "wasm32-wasi":
             trip = "wasm32-wasip1"
